Source/Microphysics/Null/NullMoistLagrangian.H

```diff
@@ -0,0 +1,54 @@
+/*! @file NullMoistLagrangian.H
+ *  \brief Contains the Lagrangian moisture model base class */
+
+#ifndef NULLMOISTLAGRANGIAN_H
+#define NULLMOISTLAGRANGIAN_H
+
+#ifdef ERF_USE_PARTICLES
+
+#include "NullMoist.H"
+
+/* forward declaration */
+class ERFPC;
```

Comment:
@asalmgren @AMLattanzi same question here: should I make `NullMoistLagrangian` a class templated on the particle container type?